templates/bastion-and-eks-cluster.yaml

```diff
@@ -1258,6 +1258,25 @@ Resources:
                   \ --clustername ${EKSClusterName}"
               timeoutSeconds: 90
       DocumentType: Command
+  
+  # Execute helm install monitoring tools
+  HelmInstallMonitoringRunScript:
+    Type: "AWS::SSM::Document"
+    Properties:
+      Content:
+        schemaVersion: "2.2"
+        description: Install Metrics Server and K8s Dashboard Helm charts
+        mainSteps:
+          - action: aws:runShellScript
+            name: helmInstallMonitoring
+            inputs:
+              workingDirectory: "/root"
+              runCommand:
+                - "chmod u+x /tmp/helmMonitoring.sh"
+                - !Sub "/tmp/helmMonitoring.sh\
+                  \ --install"
+              timeoutSeconds: 90
```

Comment:
As we're adding another command we also need to increase the timeout value for the Lambda that calls all the SSM i.e. the value on this line: https://github.com/Alfresco/acs-deployment-aws/blob/cfc70ace2b461108448e33942971d72574386ab6/templates/bastion-and-eks-cluster.yaml#L1288